src/Cli/dotnet/Commands/Test/IPC/NamedPipeServer.cs

```diff
@@ -87,6 +57,10 @@ public async Task WaitConnectionAsync(CancellationToken cancellationToken)
                 }
                 catch (Exception ex)
                 {
+                    // TODO: it might be better idea to let this exception bubble to
```

Comment:
@mariam-abdulla, what Youssef proposes sounds like a much better way to handle this. Do you agree and can you implement it?